pom.xml

```diff
@@ -47,6 +45,25 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <!-- Causes a weird ClassDefNotFound exception when this is not present -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>18.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>co.aikar</groupId>
+            <artifactId>acf-jda</artifactId>
+            <version>0.5.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.10.1</version>
```

Comment:
Why was this added and why would anyone want to use joda-time in 2019 when the creator wrote the java.time API?

Reply:
I need to be able to turn values like `5h10m30s` into actual time objects... Googling around it resulted in joda-time. I guess I can achieve the same without it then?

Reply:
java.time.Period/Duration.parse("PT5h10m30s")
it accepts the iso-idk format for periods.